### DISCUSSION TITLE
Big green bar on bottom of my screen whenever an error occurs

### DISCUSSION THREAD

**Discussion by bernarenko** [kovidgoyal/kitty#5348] [Q&A] [answered]
I'm not sure what is happening if it is a Kitty problem or what. 
Recently whenever an "alert" could've occur, a big green bar appers in the bottom of my screen. I'm not sure what might be causing it, but it just happens on Kitty.
I tried reinstalling it and use it with its default configuration(no .config/kitty files), but it still happens. 

**Screenshots**
![image](https://user-images.githubusercontent.com/20348110/182762043-274050b1-3ec7-427d-8fb2-7f0ebbe61370.png)

**Environment details**
```
kitty 0.21.2 created by Kovid Goyal
Linux Bernarenko2204U 5.15.0-43-generic #46-Ubuntu SMP Tue Jul 12 10:30:17 UTC 2022 x86_64
Ubuntu 22.04.1 LTS \n \l
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
Running under:X11
Loaded config files:
  /etc/xdg/kitty/kitty.conf

Config options different from defaults:
update_check_interval 0.0

**Comment by kovidgoyal** [comment, marked as answer]
kitty does not draw any big green bars. That's almost certinaly a bug in
your window manager/desktop environment when kitty tells the window
manager its window needs attention. https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.window_alert_on_bell

**Reply by bernarenko** [reply]
> kitty does not draw any big green bars. That's almost certinaly a bug in your window manager/desktop environment when kitty tells the window manager its window needs attention. https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.window_alert_on_bell

You were right! I simulated it in other programs and the same thing happened.
I disabled my Plank theme and it didn't occur anymore. So my choices are:
``window_alert_on_bell no``
or
changing my plank theme.

Thank you for helping me with this! Keep up the good work!
Now I'll let the plank theme creator know that it is happening!
